src/components/screens/ReleasesScreen/IframeReleasesScreen.js

```diff
@@ -22,24 +27,38 @@ const Content = styled.div`
   }
 `;
 
-function IframeReleasesScreen({ data }) {
+function IframeReleasesScreen({ data, ...rest }) {
   const {
     currentPage: {
       body,
       frontmatter: { title },
     },
   } = data;
+  const wrapperRef = useRef();
+
+  useEffect(() => {
+    if (wrapperRef && wrapperRef.current) {
+      const links = wrapperRef.current.querySelectorAll('a');
+      links.forEach((link) => {
+        /* eslint-disable no-param-reassign */
+        link.target = '_blank';
+        link.rel = 'noopener noreferrer';
+        /* eslint-enable no-param-reassign */
+      });
+    }
+  }, [wrapperRef]);
```

Comment:
I realized some of the links in the iframe open external articles on places like Medium. Well, those places don't always let you iframe their content, so it seems like all links in this view should open in a new tab.